virttest/utils_test/libvirt.py

```diff
@@ -3742,3 +3742,27 @@ def check_dumpxml(vm, content, err_ignore=False):
                 raise exceptions.TestFail("Expected '%s' was not found in "
                                           "%s's xml" % (content, vm.name))
     return True
+
+
+def clean_up_nwfilter_binding():
+    """
+    Clean up existed nwfilter binding
+
+    """
+    cmd_result = virsh.nwfilter_binding_list(debug=True)
+    binding_list = cmd_result.stdout_text.strip().splitlines()
+    binding_list = binding_list[2:]
+    result = []
+    # If binding list is not empty.
+    if binding_list:
+        for line in binding_list:
+            # Split on whitespace, assume 1 column
+            linesplit = line.split(None, 1)
+            result.append(linesplit[0])
+    logging.info("existed nwfilter binding list is: %s", result)
+    for binding_uuid in result:
+        try:
+            virsh.nwfilter_binding_delete(binding_uuid)
```

Comment:
I think it should add ignore_status=False?

Reply:
updated